frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts

```diff
@@ -82,7 +88,7 @@ describe('Add/remove disks and NICs on respective VM pages', () => {
     VM_ACTIONS_TIMEOUT_SECS,
   );
 
-  it('ID(CNV-1722) NIC cannot be added twice using one net-attach-def', async () => {
+  xit('BZ(1841051) ID(CNV-1722) NIC cannot be added twice using one net-attach-def', async () => {
```

Comment:
This test case should be removed and deleted or disabled in polarion, it's not relevant anymore
https://bugzilla.redhat.com/show_bug.cgi?id=1831825

Reply:
Change it to test the NAD can be used twice.